addons/l10n_my_ubl_pint/views/report_invoice.xml

```diff
@@ -0,0 +1,68 @@
+            </li>
+        </xpath>
+    </template>
+
+    <template id="l10n_my_ubl_pint_external_layout_bold" inherit_id="web.external_layout_bold">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <li t-if="company.sst_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                SST: <span t-field="company.sst_registration_number"/>
+            </li>
+            <li t-if="company.ttx_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                TTx: <span t-field="company.ttx_registration_number"/>
+            </li>
+        </xpath>
+    </template>
+
+    <template id="l10n_my_ubl_pint_external_layout_boxed" inherit_id="web.external_layout_boxed">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <li t-if="company.sst_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                SST: <span t-field="company.sst_registration_number"/>
+            </li>
+            <li t-if="company.ttx_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                TTx: <span t-field="company.ttx_registration_number"/>
+            </li>
+        </xpath>
+    </template>
+
+    <template id="l10n_my_ubl_pint_external_layout_striped" inherit_id="web.external_layout_striped">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
+            <li t-if="company.sst_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                SST: <span t-field="company.sst_registration_number"/>
+            </li>
+            <li t-if="company.ttx_registration_number and company.account_fiscal_country_id.code == 'MY'">
+                TTx: <span t-field="company.ttx_registration_number"/>
+            </li>
+        </xpath>
+    </template>
+
+    <!-- add the SST to the partner -->
+    <template id="l10n_my_report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[@id='partner_vat_address_not_same_as_shipping']" position="after">
+            <div t-if="o.partner_id.sst_registration_number and o.partner_id.country_code == 'MY'">
+                SST: <span t-field="o.partner_id.sst_registration_number"/>
+            </div>
+        </xpath>
+
+        <xpath expr="//div[@id='partner_vat_address_same_as_shipping']" position="after">
+            <div t-if="o.partner_id.sst_registration_number and o.partner_id.country_code == 'MY'">
+                SST: <span t-field="o.partner_id.sst_registration_number"/>
+            </div>
+        </xpath>
+
+        <xpath expr="//div[@id='partner_vat_no_shipping']" position="after">
+            <div t-if="o.partner_id.sst_registration_number and o.partner_id.country_code == 'MY'">
+                SST: <span t-field="o.partner_id.sst_registration_number"/>
+            </div>
+        </xpath>
+    </template>
```

Comment:
It's a pity we cannot do `//div[@id=('X', 'Y', 'Z')]` or `//div[@id='X' or @id='Y' or @id='Z']` (I tried both :/)

Reply:
(you don't have to do anything :D, it's just a side note)